Haskell/syntax_test_haskell.hs

```diff
@@ -93,21 +102,28 @@
 --
    a a = (+) a 2
 --     ^ keyword.operator.haskell
---       ^^^ entity.name.function.infix.haskell
+--       ^^^ variable.function.infix.haskell
+--        ^ keyword.operator.haskell
 --             ^ constant.numeric.integer.decimal.haskell
    a a = (-) a 2
 --     ^ keyword.operator.haskell
---       ^^^ entity.name.function.infix.haskell
+--       ^^^ variable.function.infix.haskell
```

Comment:
Should the `variable.function` scope *really* cover the whole thing? I generally think of the parentheses more as operators in this context (similar to the backticks in the ascii case) and the contents as the actual function.

Reply:
Same problem as https://github.com/sublimehq/Packages/pull/2145#discussion_r349674489.